### ISSUE TITLE
List doesn't work when there's no spacing between the above text and the list

### ISSUE THREAD

**Issue by S5E5S** [Automattic/Simplenote-United#138]
When I put a list under a text without a line break, the list just doesn't work
**Editor :**
![Screenshot_2022-05-14-22-12-36-27_dd6b2219cb0a90b8a293feeeaec0487a.jpg](https://user-images.githubusercontent.com/72394153/168434915-f29c021c-a5ad-4209-b796-86dc61d0c57c.jpg)
**Preview :**
Without double space after the "Text" :
![Screenshot_2022-05-30-19-09-38-58_dd6b2219cb0a90b8a293feeeaec0487a.jpg](https://user-images.githubusercontent.com/72394153/170989822-257a04e5-50f8-4d6e-b8b8-c885078f4071.jpg)
With double space after the "Text" :
![Screenshot_2022-05-30-19-09-45-65_dd6b2219cb0a90b8a293feeeaec0487a.jpg](https://user-images.githubusercontent.com/72394153/170990120-334e8fea-a5d6-4a75-a79f-6ab583d62309.jpg)
***
And when there's a line break between the text and the list, it works
**Editor :**
![Screenshot_2022-05-14-22-12-36-27_dd6b2219cb0a90b8a293feeeaec0487a~2.jpg](https://user-images.githubusercontent.com/72394153/168435097-58340efe-76f2-4ff8-908e-94cd273ecb3b.jpg)
**Preview :**
![Screenshot_2022-05-14-22-12-38-23_dd6b2219cb0a90b8a293feeeaec0487a~3.jpg](https://user-images.githubusercontent.com/72394153/168435065-de8af77d-1ac9-4046-9cf6-e2efd17aaedb.jpg)
Using double spaces also didn't solve the problem.
> Notice the spacing between the text and the list.
***
Using header instead of text will works just fine, it even eliminates the spacing problem
**Editor :**
![Screenshot_2022-05-14-22-12-36-27_dd6b2219cb0a90b8a293feeeaec0487a~3.jpg](https://user-images.githubusercontent.com/72394153/168435654-31648b90-896e-450f-831a-2bea87fa0f1e.jpg)
**Preview :**
![Screenshot_2022-05-14-22-12-38-23_dd6b2219cb0a90b8a293feeeaec0487a~4.jpg](https://user-images.githubusercontent.com/72394153/168435682-c77b692b-66de-4087-9096-1330c1e25b7a.jpg)
***
To summarize the problem :
1. List doesn't works without line breaks when there's text above it
2. Adding a line break between the text and the list would create a wide spacing (if the text and the list are related, the spacing would make it less obvious that it's related)
